jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/Annotator.java

```diff
@@ -160,6 +160,20 @@
      *         properties'.
      */
     boolean isAdditionalPropertiesSupported();
+    
+    /**
+     * Add the <code>@JsonFormat</code> annotation to mark a java.util.Date field
+     * 
+     * @param field
+     *            the field that contains data that will be serialized
+     * @param clazz
+     *            the owner of the field (class to which the field belongs)
+     * @param propertyName
+     *            the name of the JSON property that this field represents
+     * @param propertyNode
+     *            the schema node defining this property
+     */
+    void jsonFormat(JFieldVar field, JDefinedClass clazz, String propertyName, JsonNode node);
```

Comment:
The implementation of this method doesn't seem to use `clazz` at all, could you remove this from the signature?